### PR TITLE
New Site Navigation

### PR DESCRIPTION
**Pull request by kevinli-work**
* Updated site navigation based on revised homepage designs
* Old header code and styles deleted
* Converted header styling to BEM

- [x] Design review
- [x] Code review

### CODE REVIEW

**Comment by gitapablabah** [comment]
It's 508tastic, looks great. Thank you!

File: src/_scss/layouts/default/header/nav/mobile/_nav.scss [hunk 1]
```diff
@@ -1,48 +1,61 @@
...
+            font-size: rem(30);
+            line-height: rem(36);
+            opacity: 0.6;
+
+            &:active, &:hover {
+                opacity: 1;
             }
         }
+
+        button.nav-button {
```
Comment (mab6bf) [inline src/_scss/layouts/default/header/nav/mobile/_nav.scss]: Two questions – 1) Should this be something like `button.mobile-nav-content__nav-button`? and 2) Is this style even being used anymore?

Comment (kevinli-work) [reply src/_scss/layouts/default/header/nav/mobile/_nav.scss]: Not being used any more, deleted

**Comment by geoffgeiling** [comment]
Design Review Complete (need to address mobile in a future sprint)

**Pull request merged**